fastlane/metadata/ru/keywords.txt

```diff
@@ -1 +1 @@
-примечания, заметка, разметка, журнал, синхронизация, список дел, список, облако, записная книжка, простой, блокнот, метка
\ No newline at end of file
+примечания, заметка, разметка, журнал, синхронизация, список дел, список, облако, записная книжка
```

Comment:
Probably worth asking in the +i18n post that they fix that in GlotPress so you don't have to fix that yourself on every future release

Reply:
I've made the suggestion in GlotPress: https://translate.wordpress.com/projects/simplenote/ios/release-notes/ru/default/